server/opts.go

```diff
@@ -178,60 +178,60 @@ type RemoteLeafOpts struct {
-	MaxAckPending int           `json:"-"`
-	Duplicates    time.Duration `json:"-"`
+	MaxAckPending int
+	Duplicates    time.Duration
 }
 
 // Options block for nats-server.
 // NOTE: This structure is no longer used for monitoring endpoints
 // and json tags are deprecated and may be removed in the future.
 type Options struct {
-	ConfigFile            string            `json:"-"`
-	ServerName            string            `json:"server_name"`
-	Host                  string            `json:"addr"`
-	Port                  int               `json:"port"`
-	ClientAdvertise       string            `json:"-"`
-	Trace                 bool              `json:"-"`
-	Debug                 bool              `json:"-"`
-	TraceVerbose          bool              `json:"-"`
-	NoLog                 bool              `json:"-"`
-	NoSigs                bool              `json:"-"`
-	NoSublistCache        bool              `json:"-"`
-	NoHeaderSupport       bool              `json:"-"`
-	DisableShortFirstPing bool              `json:"-"`
-	Logtime               bool              `json:"-"`
-	MaxConn               int               `json:"max_connections"`
-	MaxSubs               int               `json:"max_subscriptions,omitempty"`
-	MaxSubTokens          uint8             `json:"-"`
-	Nkeys                 []*NkeyUser       `json:"-"`
-	Users                 []*User           `json:"-"`
-	Accounts              []*Account        `json:"-"`
-	NoAuthUser            string            `json:"-"`
-	SystemAccount         string            `json:"-"`
-	NoSystemAccount       bool              `json:"-"`
-	Username              string            `json:"-"`
-	Password              string            `json:"-"`
-	Authorization         string            `json:"-"`
-	PingInterval          time.Duration     `json:"ping_interval"`
-	MaxPingsOut           int               `json:"ping_max"`
-	HTTPHost              string            `json:"http_host"`
-	HTTPPort              int               `json:"http_port"`
-	HTTPBasePath          string            `json:"http_base_path"`
-	HTTPSPort             int               `json:"https_port"`
-	AuthTimeout           float64           `json:"auth_timeout"`
-	MaxControlLine        int32             `json:"max_control_line"`
-	MaxPayload            int32             `json:"max_payload"`
-	MaxPending            int64             `json:"max_pending"`
-	Cluster               ClusterOpts       `json:"cluster,omitempty"`
-	Gateway               GatewayOpts       `json:"gateway,omitempty"`
-	LeafNode              LeafNodeOpts      `json:"leaf,omitempty"`
-	JetStream             bool              `json:"jetstream"`
-	JetStreamMaxMemory    int64             `json:"-"`
-	JetStreamMaxStore     int64             `json:"-"`
-	JetStreamDomain       string            `json:"-"`
-	JetStreamExtHint      string            `json:"-"`
-	JetStreamKey          string            `json:"-"`
-	JetStreamLimits       JSLimitOpts       `json:"-"`
```

Comment:
Arg... too bad that this causes so much "difference" in the diff because of the formatting. But ultimately, we should remove those tags anyway.

Reply:
that's what I thought too, moving this field to the end seems worse though.